### PR TITLE
LPS-131759

### PR DESCRIPTION
**Pull request by gleicelisbino**
Related Issue: https://issues.liferay.com/browse/LPS-131759

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path [hunk 1]
```diff
@@ -373,7 +373,7 @@
 </tr>
 <tr>
     <td>SIDEBAR_MENUITEM_DUPLICATE_FIELD</td>
-    <td>//a[@title='Duplicate Field'] | //button[text()='Duplicate Field']</td>
+    <td>//a[@title='Duplicate'] | //button[text()='Duplicate']</td>
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path:376]: Good job, @gleicelisbino! :slightly_smiling_face:

**Comment by natocesarrego** [comment]
@gleicelisbino, can you please revert what I did on this [commit](https://github.com/liferay/liferay-portal/commit/005ffb345ff764026a051bac999eed1cec41a98f#diff-3333a7d1f44a242ced4153e04d8f7790be945ae4d31d30c8d5880d7a13564082) just for the test `AddFormWithDuplicateTextFields`?

I mean, can you please revert the following?

image

Doing it, this pull will fix the test and also add it back to the appropriated test suites. :slightly_smiling_face:

**Comment by natocesarrego** [comment]
Once you enable this test on your local branch, you can run the command `git push -f origin LPS-131759` to reflect this change here. :wink:

I'll wait this action before forward this pull to Brian.

Thanks in advance.

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
I'm forwarding this pull because Renato has already reviewed it and was just waiting for this [change](https://github.com/natocesarrego/liferay-portal/pull/1033/commits/328ad46dabbd2ddb840a37bf9c77f4a31a2f916e#diff-3333a7d1f44a242ced4153e04d8f7790be945ae4d31d30c8d5880d7a13564082R525-R529) to forward it.

Thanks @gleicelisbino !

/cc @natocesarrego

**Comment by liferay-forms** [comment]
ci:test:relevant

**Pull request closed without merging**